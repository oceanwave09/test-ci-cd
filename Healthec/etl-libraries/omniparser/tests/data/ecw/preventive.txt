apuid|RecordID|PatientId|PatientAccountNo|encounterId|EncounterDate|ProviderNPI|Ecwstructid|Question|QuestionID|Answers|AnswerID|ItemName|ItemID|CategoryName|CategoryID|MainCategoryName|MainCategoryID
280807|280807.17502.2474676.940|17502|17502|2474676|2022-09-29|1124004056|1078|Counseling for nutrition provided|940|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2507258.940|17502|17502|2507258|2022-10-06|1124004056|1078|Counseling for nutrition provided|940|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2482639.940|17502|17502|2482639|2022-10-13|1841223997|1078|Counseling for nutrition provided|940|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2398600.940|17502|17502|2398600|2022-10-17|1841223997|1078|Counseling for nutrition provided|940|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2542640.940|17502|17502|2542640|2022-10-17|1235225871|1078|Counseling for nutrition provided|940|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2430125.940|17502|17502|2430125|2022-10-18|1841223997|1078|Counseling for nutrition provided|940|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2398607.940|17502|17502|2398607|2022-10-18|1841223997|1078|Counseling for nutrition provided|940|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2477419.940|17502|17502|2477419|2022-10-19|1841223997|1078|Counseling for nutrition provided|940|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2550208.940|17502|17502|2550208|2022-10-18|1235225871|1078|Counseling for nutrition provided|940|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2373080.940|17502|17502|2373080|2022-10-19|1841223997|1078|Counseling for nutrition provided|940|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2527710.940|17502|17502|2527710|2022-10-19|1124004056|1078|Counseling for nutrition provided|940|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2527907.940|17502|17502|2527907|2022-10-19|1124004056|1078|Counseling for nutrition provided|940|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2528510.940|17502|17502|2528510|2022-10-20|1124004056|1078|Counseling for nutrition provided|940|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2536566.940|17502|17502|2536566|2022-10-21|1124004056|1078|Counseling for nutrition provided|940|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2540961.940|17502|17502|2540961|2022-10-21|1124004056|1078|Counseling for nutrition provided|940|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2539951.940|17502|17502|2539951|2022-10-21|1124004056|1078|Counseling for nutrition provided|940|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2398600.941|17502|17502|2398600|2022-10-17|1841223997|1079|Counseling for physical activity provided|941|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2542640.941|17502|17502|2542640|2022-10-17|1235225871|1079|Counseling for physical activity provided|941|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2430125.941|17502|17502|2430125|2022-10-18|1841223997|1079|Counseling for physical activity provided|941|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2398607.941|17502|17502|2398607|2022-10-18|1841223997|1079|Counseling for physical activity provided|941|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2477419.941|17502|17502|2477419|2022-10-19|1841223997|1079|Counseling for physical activity provided|941|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2550208.941|17502|17502|2550208|2022-10-18|1235225871|1079|Counseling for physical activity provided|941|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2373080.941|17502|17502|2373080|2022-10-19|1841223997|1079|Counseling for physical activity provided|941|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2540961.941|17502|17502|2540961|2022-10-21|1124004056|1079|Counseling for physical activity provided|941|Yes||Communication to patient:|298856|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2531144.828|17502|17502|2531144|2022-10-17|1396385019|263|Patient counselled on the dangers of tobacco use and urged to quit.|828|10/17/2022|| Smoking/Smokeless Tobacco|10664|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2535352.828|17502|17502|2535352|2022-10-18|1093238222|263|Patient counselled on the dangers of tobacco use and urged to quit.|828|10/18/2022|| Smoking/Smokeless Tobacco|10664|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2495288.828|17502|17502|2495288|2022-10-20|1366565202|263|Patient counselled on the dangers of tobacco use and urged to quit.|828|01/25/2021|| Smoking/Smokeless Tobacco|10664|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2521413.828|17502|17502|2521413|2022-10-19|1841223997|263|Patient counselled on the dangers of tobacco use and urged to quit.|828|10/19/2022|| Smoking/Smokeless Tobacco|10664|Counseling|10663|Preventive Medicine|10643
280807|280807.17502.2515680.828|17502|17502|2515680|2022-10-20|1235225871|263|Patient counselled on the dangers of tobacco use and urged to quit.|828|08/27/2020|| Smoking/Smokeless Tobacco|10664|Counseling|10663|Preventive Medicine|10643
