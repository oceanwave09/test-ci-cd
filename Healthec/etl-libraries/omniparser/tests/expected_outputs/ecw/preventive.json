[
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1078",
		"encounter_date": "2022-09-29",
		"encounter_id": "2474676",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "7c9fec7daf05ccbea5affb64b1d59e72",
		"question": "Counseling for nutrition provided",
		"question_id": "940",
		"record_id": "280807.17502.2474676.940",
		"row_id": "8e44d2ad-3891-477c-8cdb-48b080262d81"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1078",
		"encounter_date": "2022-10-06",
		"encounter_id": "2507258",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "7c9fec7daf05ccbea5affb64b1d59e72",
		"question": "Counseling for nutrition provided",
		"question_id": "940",
		"record_id": "280807.17502.2507258.940",
		"row_id": "c339eee0-53f6-4bef-a81e-155b3b1df59e"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1078",
		"encounter_date": "2022-10-13",
		"encounter_id": "2482639",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Counseling for nutrition provided",
		"question_id": "940",
		"record_id": "280807.17502.2482639.940",
		"row_id": "26ee9892-2510-4c40-9cf0-9091f8002a18"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1078",
		"encounter_date": "2022-10-17",
		"encounter_id": "2398600",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Counseling for nutrition provided",
		"question_id": "940",
		"record_id": "280807.17502.2398600.940",
		"row_id": "71a75df6-8fd9-4596-adb1-eb02ba08678e"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1078",
		"encounter_date": "2022-10-17",
		"encounter_id": "2542640",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "3a0d3850d7f4164bf6c5df8ca687c62a",
		"question": "Counseling for nutrition provided",
		"question_id": "940",
		"record_id": "280807.17502.2542640.940",
		"row_id": "58b3d44b-1620-42ff-b552-0e22d1d80d88"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1078",
		"encounter_date": "2022-10-18",
		"encounter_id": "2430125",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Counseling for nutrition provided",
		"question_id": "940",
		"record_id": "280807.17502.2430125.940",
		"row_id": "217773e3-17d9-477d-903f-5bddb285715c"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1078",
		"encounter_date": "2022-10-18",
		"encounter_id": "2398607",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Counseling for nutrition provided",
		"question_id": "940",
		"record_id": "280807.17502.2398607.940",
		"row_id": "204d4c08-0f91-4849-9f7d-b8b083986ff2"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1078",
		"encounter_date": "2022-10-19",
		"encounter_id": "2477419",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Counseling for nutrition provided",
		"question_id": "940",
		"record_id": "280807.17502.2477419.940",
		"row_id": "0b495c16-6d19-45b4-8ae7-c4f59486fe60"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1078",
		"encounter_date": "2022-10-18",
		"encounter_id": "2550208",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "3a0d3850d7f4164bf6c5df8ca687c62a",
		"question": "Counseling for nutrition provided",
		"question_id": "940",
		"record_id": "280807.17502.2550208.940",
		"row_id": "4cc6528e-e4b8-47ab-9966-7b6117c7daf2"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1078",
		"encounter_date": "2022-10-19",
		"encounter_id": "2373080",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Counseling for nutrition provided",
		"question_id": "940",
		"record_id": "280807.17502.2373080.940",
		"row_id": "e19d8a84-ef41-4083-8d9a-80df35fb5185"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1078",
		"encounter_date": "2022-10-19",
		"encounter_id": "2527710",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "7c9fec7daf05ccbea5affb64b1d59e72",
		"question": "Counseling for nutrition provided",
		"question_id": "940",
		"record_id": "280807.17502.2527710.940",
		"row_id": "ba4cd1e4-2bc7-42c8-83f2-af18634aa33a"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1078",
		"encounter_date": "2022-10-19",
		"encounter_id": "2527907",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "7c9fec7daf05ccbea5affb64b1d59e72",
		"question": "Counseling for nutrition provided",
		"question_id": "940",
		"record_id": "280807.17502.2527907.940",
		"row_id": "8ede40d2-6627-4ab8-b2a9-569f2ba06b6e"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1078",
		"encounter_date": "2022-10-20",
		"encounter_id": "2528510",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "7c9fec7daf05ccbea5affb64b1d59e72",
		"question": "Counseling for nutrition provided",
		"question_id": "940",
		"record_id": "280807.17502.2528510.940",
		"row_id": "27153082-5772-4183-b831-124aed821cdd"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1078",
		"encounter_date": "2022-10-21",
		"encounter_id": "2536566",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "7c9fec7daf05ccbea5affb64b1d59e72",
		"question": "Counseling for nutrition provided",
		"question_id": "940",
		"record_id": "280807.17502.2536566.940",
		"row_id": "cc23a9d5-4260-4687-b112-a04c55a469ef"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1078",
		"encounter_date": "2022-10-21",
		"encounter_id": "2540961",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "7c9fec7daf05ccbea5affb64b1d59e72",
		"question": "Counseling for nutrition provided",
		"question_id": "940",
		"record_id": "280807.17502.2540961.940",
		"row_id": "dd55fd49-2ce6-4695-a9a4-c8d311b5c801"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1078",
		"encounter_date": "2022-10-21",
		"encounter_id": "2539951",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "7c9fec7daf05ccbea5affb64b1d59e72",
		"question": "Counseling for nutrition provided",
		"question_id": "940",
		"record_id": "280807.17502.2539951.940",
		"row_id": "a67b514b-09da-4e30-a4cf-3b0a10155f64"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1079",
		"encounter_date": "2022-10-17",
		"encounter_id": "2398600",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Counseling for physical activity provided",
		"question_id": "941",
		"record_id": "280807.17502.2398600.941",
		"row_id": "866b546a-7c64-435d-9809-b8560cf8db05"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1079",
		"encounter_date": "2022-10-17",
		"encounter_id": "2542640",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "3a0d3850d7f4164bf6c5df8ca687c62a",
		"question": "Counseling for physical activity provided",
		"question_id": "941",
		"record_id": "280807.17502.2542640.941",
		"row_id": "2410c7e7-402a-4b5c-b1b9-5194acf15320"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1079",
		"encounter_date": "2022-10-18",
		"encounter_id": "2430125",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Counseling for physical activity provided",
		"question_id": "941",
		"record_id": "280807.17502.2430125.941",
		"row_id": "5ff00a80-3e4a-4e09-b459-c49389fbf3ba"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1079",
		"encounter_date": "2022-10-18",
		"encounter_id": "2398607",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Counseling for physical activity provided",
		"question_id": "941",
		"record_id": "280807.17502.2398607.941",
		"row_id": "cfc27f70-8171-4280-8600-8261b08d0e46"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1079",
		"encounter_date": "2022-10-19",
		"encounter_id": "2477419",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Counseling for physical activity provided",
		"question_id": "941",
		"record_id": "280807.17502.2477419.941",
		"row_id": "efb1d8db-e13c-4037-81aa-d3d9048c0f43"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1079",
		"encounter_date": "2022-10-18",
		"encounter_id": "2550208",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "3a0d3850d7f4164bf6c5df8ca687c62a",
		"question": "Counseling for physical activity provided",
		"question_id": "941",
		"record_id": "280807.17502.2550208.941",
		"row_id": "358c26c4-b7e5-4d9c-ad80-6775282e3aee"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1079",
		"encounter_date": "2022-10-19",
		"encounter_id": "2373080",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Counseling for physical activity provided",
		"question_id": "941",
		"record_id": "280807.17502.2373080.941",
		"row_id": "a7f64886-6230-41e7-9942-469b3d601901"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "1079",
		"encounter_date": "2022-10-21",
		"encounter_id": "2540961",
		"item_id": "298856",
		"item_name": "Communication to patient:",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "7c9fec7daf05ccbea5affb64b1d59e72",
		"question": "Counseling for physical activity provided",
		"question_id": "941",
		"record_id": "280807.17502.2540961.941",
		"row_id": "057dc495-17db-483c-885c-3154ec11fd29"
	},
	{
		"answers": "10/17/2022",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "263",
		"encounter_date": "2022-10-17",
		"encounter_id": "2531144",
		"item_id": "10664",
		"item_name": "Smoking/Smokeless Tobacco",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "1c369e06490b6b2cc59107d724bc80c8",
		"question": "Patient counselled on the dangers of tobacco use and urged to quit.",
		"question_id": "828",
		"record_id": "280807.17502.2531144.828",
		"row_id": "2fb11b78-b99d-441a-bd0d-d6dad0062183"
	},
	{
		"answers": "10/18/2022",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "263",
		"encounter_date": "2022-10-18",
		"encounter_id": "2535352",
		"item_id": "10664",
		"item_name": "Smoking/Smokeless Tobacco",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "e862212969680d9f64910c9b6b6a38d5",
		"question": "Patient counselled on the dangers of tobacco use and urged to quit.",
		"question_id": "828",
		"record_id": "280807.17502.2535352.828",
		"row_id": "65449c44-84f6-4d8a-829a-1348cc332a5e"
	},
	{
		"answers": "01/25/2021",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "263",
		"encounter_date": "2022-10-20",
		"encounter_id": "2495288",
		"item_id": "10664",
		"item_name": "Smoking/Smokeless Tobacco",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "073e509e52f867fad2fef49277008f4a",
		"question": "Patient counselled on the dangers of tobacco use and urged to quit.",
		"question_id": "828",
		"record_id": "280807.17502.2495288.828",
		"row_id": "b81ca31a-482f-4a79-b2f6-f4555bea78dd"
	},
	{
		"answers": "10/19/2022",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "263",
		"encounter_date": "2022-10-19",
		"encounter_id": "2521413",
		"item_id": "10664",
		"item_name": "Smoking/Smokeless Tobacco",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Patient counselled on the dangers of tobacco use and urged to quit.",
		"question_id": "828",
		"record_id": "280807.17502.2521413.828",
		"row_id": "593f563c-56f3-4e3b-8177-e9b92c3b6728"
	},
	{
		"answers": "08/27/2020",
		"apuid": "280807",
		"category_id": "10663",
		"category_name": "Counseling",
		"ecw_struct_id": "263",
		"encounter_date": "2022-10-20",
		"encounter_id": "2515680",
		"item_id": "10664",
		"item_name": "Smoking/Smokeless Tobacco",
		"main_category_id": "10643",
		"main_category_name": "Preventive Medicine",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "3a0d3850d7f4164bf6c5df8ca687c62a",
		"question": "Patient counselled on the dangers of tobacco use and urged to quit.",
		"question_id": "828",
		"record_id": "280807.17502.2515680.828",
		"row_id": "4ac6abb0-456f-4819-aa3a-422fbf6f62bc"
	}
]
